db/seeds.rb

```diff
@@ -1,7 +1,17 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Category.create!([
+  {name: 'Lenses'},
+  {name: 'Bodies'},
+  {name: 'Accessories'}
+  ])
+
+Item.create!([
+  {name: 'Canon EF 70-200mm f/2.8L', description: 'Two mode optical image stabilization', price: 14, image_url: 'http://www.bhphotovideo.com/images/images1000x1000/Canon_2751B002_EF_70_200mm_f_2_8L_IS_680103.jpg', category_id: 1},
+  {name: 'Canon EF 16-35mm f/2.8L', description: 'Ultra Wide Angle', price: 13, image_url: 'http://www.bhphotovideo.com/images/images2500x2500/Canon_1910B002AA_EF_16_35mm_f_2_8L_II_486708.jpg', category_id: 1},
```

Comment:
Line is too long. [210/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.